src/zif_abapgit_definitions.intf.abap

```diff
@@ -25,8 +25,8 @@ INTERFACE zif_abapgit_definitions
   TYPES:
     BEGIN OF ty_file.
       INCLUDE TYPE ty_file_signature.
-  TYPES: data TYPE xstring,
-         END OF ty_file .
+      TYPES: data TYPE xstring,
```

Comment:
try this:
```suggestion
  TYPES: data TYPE xstring,
```

Reply:
I changed indentation for these TYPES, but I think that they were in correct position, they're inside another TYPE